index1.html

```diff
@@ -0,0 +1,267 @@
+          <li>2015</li>
+        </ul>
+        <p class="work-details">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="work-tools">
+          <li>html</li>
+          <li>css</li>
+          <li>Javascript</li>
+        </ul>
+        <button class="btn">See Project</button>
+      </div>
+    </div>
+    <div class="project">
+      <div class="work-img">
+        <img src="/images1/4.svg" alt="tonic image" />
+      </div>
+      <div class="work-description">
+        <h2 class="work-title">Multi-Post Stories</h2>
+        <ul class="canopy">
+          <li>canopy</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+        <p class="work-details">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="work-tools">
+          <li>html</li>
+          <li>css</li>
+          <li>Javascript</li>
+        </ul>
+        <button class="btn">See Project</button>
+      </div>
+    </div>
+  </section>
+
+  <section id="about">
+    <div class="about-container">
+      <h2 class="myself">About Myself</h2>
+      <p class="about-details">
+        Hello I’m a software developer! I can help you build a product ,
+        feature or website. Look through some of my work and experience! If
+        you like what you see and have a project you need coded, don’t
+        hesitate to contact me.
+      </p>
+      <h4 class="connect">LET'S CONNECT</h4>
+      <ul class="socials">
+        <li>
+          <a href="https://github.com/Eccb7">
+            <i><img class="icons" src="/images1/github.svg" alt="" /></i>
+          </a>
+        </li>
+        <li>
+          <a href="https://www.linkedin.com/in/briton-otieno-ba6591244/">
+            <i><img class="icons" src="/images1/Linkedin.svg" alt="" /></i>
+          </a>
+        </li>
+        <li>
+          <a href="">
+            <i><img class="icons" src="/images1/peace.svg" alt="" /></i>
+          </a>
+        </li>
+        <li>
+          <a href="https://twitter.com/briton_otieno4">
+            <i><img class="icons" src="/images1/twitter.svg" alt="" /></i>
+          </a>
+        </li>
+        <li>
+          <a href="">
+            <i><img class="icons" src="/images1/Vector.svg" alt="" /></i>
+          </a>
+        </li>
+      </ul>
+      <button class="btn resume">Get my resume</button>
+    </div>
+    <ul class="skills">
+      <li class="languages">
+        <div class="languages skill-group">
+          <p class="title">languages</p>
+          <div class="union-icon">
+            <img class="active" src="/images1/Union.svg" alt="" />
+          </div>
+        </div>
+        <ul class="language-list">
+          <li>
+            <p><img src="/images1/javascript.svg" alt="" /></p>
+            <p class="lang-name">JavaScript</p>
+          </li>
+          <li>
```

Comment:
- [ ] As you can see even here your  social media icons are not displaying :point_down:  kindly fix this so that your project will look more professional :wink: 
![Screenshot from 2023-05-05 22-33-51](https://user-images.githubusercontent.com/88894525/236565486-eebc7cf7-5b22-45c2-b8fb-a180726eaf44.png)